test/config_parser_SUITE_data/s2s_only.cfg

```diff
@@ -0,0 +1,17 @@
+{hosts, [
+    "localhost",
+    "grubalubadubdub"
```

Comment:
is it valid host name? :D

Reply:
For the purpose of this task it may very well be